Windows/windows/ingest/parser.yml

```diff
@@ -358,6 +359,49 @@ stages:
             {% for component in parsed_message_xml.result.AuditBase.ContextComponents.Component   %}{%if "ProxyServer" in component%}{{component["ProxyServer"].strip()}}{% endif %}{% endfor %}
         filter: "{{json.event.SourceName == 'AD FS Auditing'}}"
 
+      - set:
+          action.properties.AdditionalActionsID: "{{ json.event.get('Additional Actions ID') }}"
+          action.properties.AdditionalActionsString: "{{ json.event.get('Additional Actions String') }}"
+
+      - set:
+          action.properties.CertIssuerName: "{{json_event.message.CertIssuerName}}"
+          action.properties.CertSerialNumber: "{{json_event.message.CertSerialNumber}}"
+          action.properties.CertThumbprint: "{{json_event.message.CertThumbprint}}"
+
+      - set:
+          action.properties.Category: "{{json_event.message.Category}}"
+          action.properties.Issuer: "{{json_event.message.Issuer}}"
+
+      - set:
+          action.properties.OldSd: "{{json_event.message.OldSd}}"
+          action.properties.NewSd: "{{json_event.message.NewSd}}"
+
+      - set:
+          action.properties.ObjectDN: "{{json_event.message.ObjectDN}}"
+          action.properties.ObjectGUID: "{{json_event.message.ObjectGUID}}"
+          action.properties.ObjectClass: "{{json_event.message.ObjectClass}}"
+
+      - set:
+          action.properties.DestinationDRA: "{{json_event.message.DestinationDRA}}"
+          action.properties.SourceDRA: "{{json_event.message.SourceDRA}}"
+          action.properties.NamingContext: "{{json_event.message.NamingContext}}"
+          action.properties.StartUSN: "{{json_event.message.StartUSN}}"
+          action.properties.EndUSN: "{{json_event.message.EndUSN}}"
+          action.properties.SessionID: "{{json_event.message.SessionID}}"
+
+      - set:
+          action.properties.NASIdentifier: "{{json_event.message.NASIdentifier}}"
+          action.properties.NASPortType: "{{json_event.message.NASPortType}}"
+          action.properties.NASPort: "{{json_event.message.NASPort}}"
+          action.properties.ClientName: "{{json_event.message.ClientName}}"
+          action.properties.ClientIPAddress: "{{json_event.message.ClientIPAddress}}"
+          action.properties.ProxyPolicyName: "{{json_event.message.ProxyPolicyName}}"
+          action.properties.NetworkPolicyName: "{{json_event.message.NetworkPolicyName}}"
+          action.properties.AuthenticationProvider: "{{json_event.message.AuthenticationProvider}}"
+          action.properties.AuthenticationServer: "{{json_event.message.AuthenticationServer}}"
+          action.properties.AuthenticationType: "{{json_event.message.AuthenticationType}}"
+          action.properties.EAPType: "{{json_event.message.EAPType}}"
+
```

Comment:
Could you move action.properties fields to the dedicated stage ?

https://github.com/SEKOIA-IO/intake-formats/blob/7df70e5ce335285ece986e0639fbdf4984dc7d51/Windows/windows/ingest/parser.yml#L295

Reply:
Thanks, good to know

Reply:
They're actually there